app/test_server.py

```diff
@@ -3,7 +3,9 @@
 import pytest
 import requests
 
+from .scrapers import scrapers, small_test
```

Comment:
import from the python `__init__` , not from the folder.

Reply:
Given the presence of the \_\_init\_\_.py file, it is better to think of __scrapers__ as a Python package rather than as a folder.  What would be the advantage of exposing the internal implementation of the scrapers package to the caller?  Doesn't that needlessly limit the flexibility of the package implementer?  Are there other instances in the wild where people write __from \_\_init\_\_.py import xyz__ ?  I have not seen this before.

Reply:
I think you should replace `.scrapers` with `scrapers` only.

Reply:
Did you try that in Python 3?  https://www.python.org/dev/peps/pep-0328/#guido-s-decision

Reply:
Nope, I am using python 2.7. 
I think [this](https://github.com/fossasia/query-server/blob/eb7c7aabf55145f94a2195c78af42b9e467d5fc8/app/server.py#L10) might also help. Shouldn't it work the same way here?